requirements_frozen.txt

```diff
@@ -108,10 +108,8 @@ git+https://github.com/mhahnenberg/sqlalchemy.git@e5f7586346ce5fafe2c182f70bf6ef
 statsd==3.1
 structlog==17.2.0
 tldextract==1.7.5
-tox==2.3.1
 uritemplate==0.6
 urllib3==1.22
-virtualenv==15.1.0
```

Comment:
This is only a dependency of tox. Tox creates an isolated virtualenv to run each command.